core-bundle/src/Resources/contao/languages/de/default.xlf

```diff
@@ -2681,6 +2681,10 @@
         <source>Close the Contao toolbar</source>
         <target>Contao-Toolbar schließen</target>
       </trans-unit>
+      <trans-unit id="MSC.copyURL">
```

Comment:
please remove the german translation from the PR, these are managed on transifex.com